### DISCUSSION TITLE
Duplicate Finder

### DISCUSSION THREAD

**Discussion by NorthernMunkey** [photoprism/photoprism#3287] [Ideas]
I think across the general userbase this would be quite a useful feature,  it's missing in google photo's but available in basic windows apps,  
Having a duplicate finder based off criteria ( file size, timestamps, exif info, jpeg comparison and or a combination of all of them) with the ability to exclude and include areas/folders.

**Comment by eljejer** [comment]
Processing of duplicates is described here (with a promise to deal with it in a future release)
https://docs.photoprism.app/user-guide/library/duplicates/

**Reply by hellomiakoda** [reply]
The current implication is insufficient. I imported quite a few folders of photos, from various places scattered about my digital live. I'm finding my PhotoPrism is full of photos that are identical, but because they have different file names or were in different folders, it's not doing anything about it. I'm going to have to go in manually and delete duplicates, or mount my server as a folder and run other software to find and remove duplicates - once I read up and understand whether or not deleting photo files will confuse PhotoPrism. It's not even stacking things that are clearly 2 identical copies of the same photo. I could understand if it doesn't stack a photo and a copy that had it's contrast changed or a filter applied, but literal duplicate copies are just being tossed in.

**Comment by cmikovits** [comment]
Is there information on how duplicate detection works? E.g. just by filename, by checksum, or "real" image analysis?

**Reply by lastzero** [reply]
https://docs.photoprism.app/user-guide/library/duplicates/
